controllers/hibp.js

```diff
@@ -67,7 +67,7 @@ async function notify (req, res) {
     }
   }
 
-  if (breachAlert.IsSpamList || breachAlert.IsFabricated || !breachAlert.IsVerified) {
+  if (breachAlert.IsSpamList || breachAlert.IsFabricated || !breachAlert.IsVerified || breachAlert.Domain === "") {
```

Comment:
Looks like this was originally changed in #1397.
Did we only change it short term for the popular PDL breach?

Reply:
Yep - this was just a short term change for PDL.

Reply:
Started https://github.com/mozilla/blurts-server/issues/1404#issuecomment-564018724 to consider making this change permanent as we improve recommendations.